### PR TITLE
feat(HelpBar): connect feedback form with feedback link

### PR DESCRIPTION
**Pull request by blegesse-git**
Inside free tier contact support, the link directing user to feedback and questions form has an empty href. 
Pr adds an `onClick` property that opens the Feedback and Questions form. 

https://user-images.githubusercontent.com/66275100/171212858-97e0329c-ebb1-46e6-ac53-2b2a5f1b4738.mov

### CODE REVIEW

File: src/support/components/FreeAccountSupportOverlay.tsx [hunk 1]
```diff
@@ -32,6 +35,15 @@ const FreeAccountSupportOverlay: FC<OwnProps> = () => {
   const {onClose} = useContext(OverlayContext)
   const isCredit250ExperienceActive = useSelector(shouldGetCredit250Experience)
 
+  const dispatch = useDispatch()
+
+  const openFeedbackOverlay = (event): void => {
+    event.preventDefault()
```
Comment (hoorayimhelping) [inline src/support/components/FreeAccountSupportOverlay.tsx:41]: I don't think the `event.preventDefault()` is necessary here. What happens when you remove it?

Also, do we want an `event` here?

**Review by hoorayimhelping — COMMENTED** [review]
Nice find! A couple of questions

File: src/support/components/FreeAccountSupportOverlay.tsx (hunk 1 above)
Comment (hoorayimhelping) [reply src/support/components/FreeAccountSupportOverlay.tsx:41]: @blegesse-git cool, looks like the move is to leave the `eventPrevent` :) (nice answer too heh)

do we need an app event like `event('helpBar.freeAccount.supportOverlay.shown')`

Comment (blegesse-git) [reply src/support/components/FreeAccountSupportOverlay.tsx:41]: haha! yes. i didn't realize i answered your question with a question. 
and i thought about that too. I think adding an event metrics there could be helpful. But that means adding event metrics for both payg and free account overlays  inside `TreeNav` component as well. Which is a-okay with me. I'll get the fix up soon

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**